app/src/pages/inside/common/infoPanel/viewTabs/viewTabs.jsx

```diff
@@ -106,49 +110,66 @@ export class ViewTabs extends Component {
         link: listViewLink,
         icon: ListIcon,
         available: true,
+        disabled: false,
+        hint: false,
       },
       {
         id: UNIQUE_ERRORS_VIEW,
         title: formatMessage(messages[UNIQUE_ERRORS_VIEW]),
-        link: listViewLink,
+        link: {},
```

Comment:
Empty link action will cause error (see the screenshot below)
![image](https://user-images.githubusercontent.com/31861805/111140737-bd723f80-8593-11eb-8717-790133ade8b8.png)
To avoid this make sense to leave previous link (`listViewLink`) here and specify `shouldDispatch={!page.disabled}` as `Link` prop (https://github.com/faceyspacey/redux-first-router-link#additional-props) or simply do not render `Link` component at all for disabled items (render just content).